system-test/system_test.go

```diff
@@ -64,7 +75,7 @@ BASE_DIR=$(pwd)
 NODEDIR=$(dirname $(dirname $(which node)))
 
 # Build and pack pprof module.
-cd {{.PprofDir}}
+cd {{.PprofNodejsDir}}
```

Comment:
In my original comment I advised against this name as I don't think it's clear enough. Could you base the name on the purpose rather than the name of module?